pkg/network/node/iptables.go

```diff
@@ -27,6 +29,39 @@ type NodeIPTables struct {
 	egressIPs map[string]string
 }
 
+// this could potentially add 13 to an iptables call, this should happen rarely
```

Comment:
missing "seconds"
and it doesn't have to be phrased like a warning, I was just saying, often when you have a retry like this, you want to be able to figure out how long it's going to be retrying for, and that's hard to do with exponential backoff, so it's helpful to have a comment like "this will retry 10 times over a period of about 13 seconds" to clarify

Reply:
done